src/react/components/pages/homepage/homePage.tsx

```
@@ -150,4 +189,16 @@ export default class HomePage extends React.Component<IHomepageProps> {
             throw new AppError(ErrorCode.ProjectDeleteError, "Error deleting project file");
         }
     }
+
+    private convertProject = async (project: IFileInfo) => {
+        const importService = new ImportService();
+        let projectJson;
+        try {
+            projectJson = await importService.convertV1(project);
+        } catch (e) {
+            throw new AppError(ErrorCode.ProjectUploadError, "Error converting v1 project file");
```

Looks like this should throw the convert error code vs. project upload error.